bcftools/1.20/Dockerfile

```diff
@@ -63,6 +65,7 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
     bzip2 \
     liblzma-dev \
     libcurl4-gnutls-dev \
```

Comment:
Do these actually need to be -dev versions with headers in the runtime container? Do plugins somehow compile/link against them at runtime or can they just be `liblzma libcurl4-gnutls`?

Reply:
Not sure about compilation & linking, but I think the correct syntax is `liblzma5` for Ubuntu Jammy: https://packages.ubuntu.com/search?suite=jammy&section=all&arch=any&keywords=liblzma&searchon=names

and `libcurl4`: https://packages.ubuntu.com/search?suite=jammy&section=all&arch=any&keywords=libcurl4&searchon=names

I believe when the samtools/bcftools/htslib dockerfiles were written we were following these instructions: https://github.com/samtools/samtools/blob/972c1889942a4f07d8f62e93330f723da919c271/INSTALL#L220